scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -382,12 +382,22 @@
             :to-make-room-for pending-job)]
     [state nil]))
 
+(defn reserve-hosts! [state preemption-decisions]
```

Comment:
Missing unit tests, it would have helped me understand the structure of `state` and `preemption-decisions`

Reply:
Nitpick: missing docstring

Reply:
Done